aksel.nav.no/website/sanity/schema/objects/templates/ExampletextBlock.tsx

```diff
@@ -13,7 +13,18 @@ export const ExampletextBlock = defineType({
       name: "title",
       type: "string",
       initialValue: "Eksempeltekst",
-      validation: (Rule) => Rule.required(),
+      validation: (Rule) =>
+        Rule.required().custom((value, context) => {
+          if (!context.document) return true;
+          const content = context.document.content as any[];
+          const blocksWithThisTitle = content
```

Comment:
If we want so fix all the cases we also need to check `expansioncard`. 
```js
if(block._type === "expansioncard"){
    const nestedBlocksWithThisTitle = content.body.filter((block) => block._type === "exampletext_block").filter((block) => block.title === value);
}

```

Expansioncard is within its own`<section />`, so the a11y validation might be different and in its own context 🤔

Reply:
Axe doesn't seem to care about nested sections, but I guess they should be unique as well? 🤔

Reply:
Never mind, it does, but we apparently exclude expansioncard from the test 😆 `.exclude("#aksel-expansioncard")`